<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room</class>
 <widget class="QWidget" name="room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(91, 91, 91);</string>
  </property>
  <widget class="QPushButton" name="PB_Play1HeadIcon">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>28</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Play2HeadIcon">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>28</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LB_VSlogo">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>28</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res/res.qrc">:/logo/vs.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_RoomID">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>房间号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Stat">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>PB_Play1HeadIcon</zorder>
  <zorder>PB_Play2HeadIcon</zorder>
  <zorder>LB_VSlogo</zorder>
  <zorder>LB_RoomID</zorder>
  <zorder>LB_Stat</zorder>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
